File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.cs
using System;

public class Person
{
    private string name;
    private int age;

    // Constructor
    public Person(string name, int age)
    {
        this.name = name;
        this.age = age;
    }

    // Method
    public void Print()
    {
        Console.WriteLine($"Name: {name}, Age: {age}");
    }
}

public class Program
{
    // Standalone function
    static int Add(int a, int b)
    {
        return a + b;
    }
}

File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.java
public class Example {
    public static class Person {
        private String name;
        private int age;

        // Constructor
        public Person(String name, int age) {
            this.name = name;
            this.age = age;
        }

        // Method
        public void print() {
            System.out.printf("Name: %s, Age: %d%n", name, age);
        }
    }

    // Standalone function (static method in Java)
    public static int add(int a, int b) {
        return a + b;
    }
}